uber/models.py

```diff
@@ -856,6 +861,7 @@ def model_mixin(cls, model):
 
 
 class Group(MagModel, TakesPaymentMixin):
+    search_key    = Column(UUID, default=lambda: str(uuid4()))
```

Comment:
Does it make any sense to:
1. Have **very** random EVENT_QR_IDs
2. Restrict Group Names (at least) from starting with EVENT_QR_ID
3. **DO BOTH**

??